src/apps/front-office/design-system/components/ui/DealsOfTheDay/Header.jsx

```diff
@@ -0,0 +1,19 @@
+import { FaAngleRight } from "react-icons/fa";
+
+const Header = ({ title, linkText, linkHref }) => {
+  return (
+    <div className="flex justify-between items-center p-4">
+      <h3 className="font-quicksand text-slate-600 leading-tight mr-8 text-2xl font-bold mb-4">
+        {title}
+      </h3>
+      <a
```

Comment:
Please, use link component and URLS object to link the pages instead of writing it statically.